content/docs/06-integrations/00-kubernetes.md

```diff
@@ -16,118 +16,593 @@ import Tooltip from "shared/components/ui/Tooltip";
 
 # Kubernetes
 
-This pack defines the properties that will be used by Spectro Cloud to bring up the Kubernetes cluster. Most of the Kubernetes hardening standards recommended by the Center for Internet Security is enabled by default.
+The Kubernetes pack defines the default properties that we use to deploy Kubernetes clusters. Most of the Kubernetes hardening standards that the Center for Internet Security (CIS) recommends are enabled by default. The Kubernetes pack supports most [cloud and data center infrastructure providers](/clusters).
 
-<WarningBox>
-Info: Only Kubernetes versions 1.19 and above is supported
-</WarningBox>
+Spectro Cloud offers Cloud Native Computing Foundation (CNCF) Kubernetes as a core infrastructure pack in Palette. We release Kubernetes updates as follows:
+
+<br />
+
+- Major versions are assessed based on the extent of changes.
+- Minor versions are provided within eight weeks of a new Kubernetes release.
+- Patch versions are provided within four weeks of a new Kubernetes release.
+
+In the period of time following a Kubernetes release, the Kubernetes pack undergoes rigorous vulnerability scans and penetration testing. 
+
+A minor Kubernetes version will be deprecated in Palette when the Kubernetes community announces the version is entering End of Life (EOL). The deprecated version will remain in that state for three months and will be disabled after a quarter and deleted from Palette in the following quarter.
+
+Below are the stages of deprecation:
+
+<br />
 
-## Supported Kubernetes versions
+- **Deprecate**: When a pack is deprecated, Palette displays a warning. You will still be able to create new cluster profiles using the pack and launch clusters using existing profiles that contain the pack.
+
+
+- **Delete**: When a pack is disabled, you will no longer be allowed to create new cluster profiles that using the pack. Palette displays a message informing you to cut over to the newer version of the pack. However, you can still deploy new clusters using existing profiles that contain the deprecated pack.
```

Comment:
Should this be disabled ? Both this and the next paragraph says Delete

Reply:
This is from the [Decprecation Schedule](https://spectrocloud.atlassian.net/wiki/spaces/ENGINEERIN/pages/1863843883/Packs+Deprecation). Leaving it as is.

![CleanShot 2023-03-28 at 11 16 22](https://user-images.githubusercontent.com/117382432/228330947-56f6e320-5263-492c-957c-0bfa37503a56.png)